sos/report/plugins/aap_containerized.py

```diff
@@ -121,5 +121,19 @@ def _get_aap_container_names(self, username):
 
     # Check and enable plugin on a AAP Containerized host
     def check_enabled(self):
-        ps = self.exec_cmd("ps --noheaders axco command")
-        return "awx-manage" in ps["output"] and "aap-gateway" in ps["output"]
+        aap_processes = [
+                'dumb-init -- /usr/bin/envoy',
+                'dumb-init -- /usr/bin/supervisord',
+                'dumb-init -- /usr/bin/launch_awx_web.sh',
+                'dumb-init -- /usr/bin/launch_awx_task.sh',
+                'pulpcore-content --name pulp-content --bind',
+                'dumb-init -- aap-eda-manage',
+            ]
+
+        ps_output = self.exec_cmd("ps --noheaders -eo args")
+
+        if ps_output['status'] == 0:
+            for process in aap_processes:
+                if process in ps_output['output']:
+                    return True
```

Comment:
You mean if *either* command from the above list is running on the system, we deduce the system runs containerized AAP, right..? (not *all* cmds, but *either* of them?)

Reply:
@pmoravec thats correct. If one of the process match then it should be enabled.